service/src/main/kotlin/fi/espoo/evaka/nekku/NekkuService.kt

```diff
@@ -99,15 +118,54 @@ class NekkuService(
             )
         }
     }
+
+    fun planNekkuOrders(dbc: Database.Connection, clock: EvakaClock) {
+        planNekkuOrderJobs(dbc, asyncJobRunner, clock.now())
+    }
+
+    fun sendNekkuOrder(dbc: Database.Connection, clock: EvakaClock, job: AsyncJob.SendNekkuOrder) {
+        if (client == null) error("Cannot send Nekku order: NekkuEnv is not configured")
+        try {
+            createAndSendNekkuOrder(
+                client,
+                dbc,
+                customerNumber = job.customerNumber,
+                groupId = job.customerGroupId,
+                unitSize = job.unitSize,
+                date = job.date,
+            )
+        } catch (e: Exception) {
+            logger.warn(e) {
+                "Failed to send meal order to Nekku: date=${job.date}, customerNumber=${job.customerNumber}, groupId=${job.customerGroupId},error=${e.localizedMessage}"
+            }
+            throw e
+        }
+    }
 }
 
 interface NekkuClient {
 
+    data class NekkuOrder(
+        val delivery_date: String,
+        val customer_number: String,
+        val group_id: String,
+        val items: List<Item>,
+        val description: String,
+    )
+
+    data class Item(val sku: String, val quantity: Int, val product_options: List<ProductOption>?)
+
+    data class ProductOption(val field_id: String, val value: String)
+
+    data class NekkuOrders(val orders: List<NekkuOrder>, val dry_run: Boolean)
```

Comment:
Poista käyttämätön parametri
```suggestion
    data class NekkuOrders(val orders: List<NekkuOrder>)
```

Reply:
Kyllä tuota parametriä käytetään. 

https://dev.cpu.fi/nekku-test/swagger/#/default/post_wp_json_wc_ceepos_nekku_v1_orders